### ISSUE TITLE
emq.conf tls-psk

### ISSUE THREAD

**Issue by PiedPieper** [emqx/emqx#2311]
In etc/emqx.conf , it says that we can use TLS-PSK with AWS : 
```
## Ciphers for TLS PSK.
## Note that 'listener.ssl.external.ciphers' and 'listener.ssl.external.psk_ciphers' cannot
## be configured at the same time.
## See 'https://tools.ietf.org/html/rfc4279#section-2'.
#bridge.aws.psk_ciphers = PSK-AES128-CBC-SHA,PSK-AES256-CBC-SHA,PSK-3DES-EDE-CBC-SHA,PSK-RC4-SHA
```
As far as I'm aware and google has shown me, AWS IoT doesn't support PSK. Am I missing something, or is this configuration snippet misplaced?

**Comment by terry-xiaoyu** [comment]
@PiedPieper The `bridge` feature is not only for bridge to `aws`, it can bridge to any IoT hub, mqtt broker or even to another emq.  The word `aws` or `asure` here is only a name and can be replaced by any thing you like. 

PSK is really a new feature of emq, actually I wrote it just one day ago and it will be included in emqx v3.1-beta2. I am glad that you've noticed it so quickly. But we have not complete the psk client side of the bridge. The `bridge.aws.psk_ciphers` is there for configuring psk-related ciphers when bridging to a remote broker, but we still need to add two more configurations to configured the psk-id and the client-side psk.

The server side is ready. You could enable PSK authentication by enabling psk_ciphers (e.g. listener.ssl.external.psk_ciphers) and loading the [emqx-psk-file plugin](https://github.com/emqx/emqx-psk-file). pskid-psk pairs can be configured in etc/psk.txt.

Test the connection using `pskid = client1` and `psk = 1234`:
```
openssl s_client -cipher 'PSK-AES128-CBC-SHA' -connect 127.0.0.1:8883 -psk_identity 'client1' -psk 31323334
```

**Comment by PiedPieper** [comment]
Well thanks a lot for adding this feature, I'm sure many will appreciate it

**Comment by Gilbert-Wong** [comment]
It has been implemented in emqx 3.1-beta2.
